main.py

```diff
@@ -138,6 +147,11 @@ def save_xml(xmlstring, outfile):
     save_setjson(combinedjson, 'spoiler')
     spoilers.write_combined_xml(combinedjson, setinfos)
     save_xml(spoilers.pretty_xml('out/spoiler.xml'), 'out/spoiler.xml')
+    cockatrice_xsd = requests.get('https://raw.githubusercontent.com/Cockatrice/Cockatrice/master/doc/cards.xsd').text
+    if verify_xml('out/spoiler.xml', cockatrice_xsd):  # check if our XML passes Cockatrice's XSD
+        print 'XML passes XSD verification'
+    else:
+        print 'XML fails XSD verification'
```

Comment:
Specify in the message that it's the XSD from Cockatrice?

What happens if location of the xsd file changes or file is not available etc.? Is there an helpful error in that case?